docs/howto/operate/add-hub.md

```diff
@@ -1,8 +1,67 @@
 # Add a new hub
 
-Adding a new hub requires that we do three things:
+## Step 1: Decide where the hub will live
 
-1. [Create a new GitHub issue for the hub](https://github.com/2i2c-org/pilot-hubs/issues/new).
-2. Create the hub via appending a new hub entry in the appropriate cluster file under
-   `config/hubs` file (see [](../../topic/config.md) for more information).
-3. Start following [team process](team-process) around operating the hubs.
+Hubs need to exist on a kubernetes cluster, so we will need to
+decide which cluster it should live in.
+
+1. For standalone hubs where we are being paid or the client has cloud
+   credits, we will need to create a cluster specifically for this hub
+   before deploying it.
+2. For hubs we are running for free, they will go into the `pilot-hubs`
+   cluster.
+3. Hubs run with the support of [cloudbank](https://www.cloudbank.org/),
+   will go in the `cloudbank` cluster.
+
+Each cluster has configuration + list of hubs it supports under
+`config/hubs/<cluster-name>.cluster.yaml`.
+
+## Step 2: Decide the template used for the hub
+
+Based on the kinda usage expected of the hub select a [hub template](../../topic/hub-templates.md)
+to be used.
+
+## Step 4: Decide the authentication provider to be used
+
+In consultation with the users, decide
+[which authentication provider](https://pilot.2i2c.org/en/latest/admin/configuration/login.html#authentication)
+the hub should use. While this can be changed later, it's a messy
+process - so we should try to get this right the first time.
+
+## Step 5: Decide the user image for the hub
+
+The default user image is present in this repository (`images/user`),
+and is geared towards simple datascience classes - with both R and
+Python. Determine if this is acceptable - if not, you need to
+make a [custom image](../configure/update-env.md) for them.
+
+## Step 6: Add hub config
+
+Add an entry for the hub in `config/hubs/<cluster-name>.cluster.yaml` for this hub.
+The following docs might be helpful:
+
+1. [Configuration Structure](../../topic/config.md)
+2. [JSON Schema for `.cluster.yaml` file](https://github.com/2i2c-org/pilot-hubs/blob/master/config/hubs/schema.yaml)
+3. [Zero to JupyterHub on k8s Docs](https://zero-to-jupyterhub.readthedocs.io/en/latest/), since ultimately
+   that is what we are configuring.
+
+## Step 7: Deploy manually and test
+
+[Deploy the hub manually](./manual-deploy.md) and test to make sure it works
```

Comment:
We have automated deployment for the pilots and cloudbank hubs... are we encouraging here to do the manual deployment instead? Or we should include also the automatic deployment and then explain the testing steps indicated below?

Reply:
Or more generally, it'd be useful to have some guide for when we should try a manual deploy vs. just auto-deploying vs. config.yaml. The manual deploy step I suspect will be a big amount of extra work per hub (and will mean that the only people that can add new hubs will also need to have the skills to manually deploy a hub), so do we _really_ need to do it each time?

Reply:
Another approach could be to tell the reader to auto-deploy the hub via editing `config.yaml`, and to then confirm the following things work as expected before telling the user representative about it?